com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/helpers/ApplicationNameSuffixAppenderTest.java

```diff
@@ -0,0 +1,57 @@
+package com.sap.cloud.lm.sl.cf.core.helpers;
+
+import org.junit.jupiter.api.BeforeEach;
+
+import com.sap.cloud.lm.sl.cf.core.model.ApplicationColor;
+import com.sap.cloud.lm.sl.common.util.TestUtil;
+import com.sap.cloud.lm.sl.common.util.Tester;
+import com.sap.cloud.lm.sl.common.util.Tester.Expectation;
+import com.sap.cloud.lm.sl.mta.handlers.v2.DescriptorParser;
+import com.sap.cloud.lm.sl.mta.model.DeploymentDescriptor;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class ApplicationNameSuffixAppenderTest {
+
+    private final Tester tester = Tester.forClass(getClass());
+
+    private DescriptorParser descriptorParser;
+
+    @BeforeEach
+    public void createParser() {
+        descriptorParser = getDescriptorParser();
```

Comment:
Can we remove this method and initialize DescriptorParser directly?

Reply:
Yes, will remove this method